packages/sanity/src/sanity.js

```diff
@@ -0,0 +1,201 @@
+const sanityClient = require('@sanity/client');
+const _ = require('lodash');
+
+const { utils } = require('@stackbit/stackbit-pull-core');
+
+/**
+ *
+ * @param projectId
+ * @param ssgType
+ * @param token
+ * @param options {{dataset: string, preview: boolean, resolveLinks: boolean, allObjects: boolean, dataFormat: string, metadata: boolean, studioUrl: string}}
+ * @returns {PromiseLike<T>}
+ */
+function pull(projectId, ssgType, token, options) {
```

Comment:
It seems that you have renamed the standard `environment` variable to `dataset`.
Will the container continue to work if parameters are different?

I think container's options have to be supported in order for it to work with the new stackbit-pull.
Currently container sends the following options.
```js
const pullOptions = _.omitBy({
  stackbitPullApiUrl: pullUrl,
  apiKey: STACKBIT_API_SECRET,
  dataFormat: 'object',
  ver: 2,
  preview: true,
  metadata: true,
  resolveLinks: false,
  environment: this.repoPublishBranch || this.repoBranch
}, _.isNil);
```

So either stackbit-pull needs additional layer that translates the parameters, or every specific cms pull (contentful-pull/sanity-pull) needs to support common option names such as `preview`/`metadata`/`resolveLinks`/`environments`